src/window/keyboard_manager.rs

```diff
@@ -68,6 +73,26 @@ impl KeyboardManager {
                 // text
                 log::trace!("{:?}", *modifiers);
                 self.modifiers = *modifiers;
+
+                #[cfg(target_os = "macos")]
+                {
+                    let ws = SETTINGS.get::<WindowSettings>();
+                    self.meta_is_pressed = match ws.input_macos_option_key_is_meta.0 {
+                        OptionAsAlt::Both => self.modifiers.state().alt_key(),
+                        OptionAsAlt::OnlyLeft => {
+                            self.modifiers.lalt_state() == ModifiersKeyState::Pressed
+                        }
+                        OptionAsAlt::OnlyRight => {
+                            self.modifiers.ralt_state() == ModifiersKeyState::Pressed
+                        }
+                        OptionAsAlt::None => false,
+                    };
+
+                    #[cfg(not(target_os = "macos"))]
+                    {
+                        self.meta_is_pressed = self.modifiers.state().alt_key();
```

Comment:
This confuses me a bit, since no one calls the alt key meta anymore. More typically the windows key is called meta. 

But I guess in this context it's ok, since Vim(Neovim) also refers to it as meta.